PackageInfo.g

```diff
@@ -91,7 +83,7 @@ PackageDoc := rec(
 
 Dependencies := rec(
   GAP := ">=4.7",
-  NeededOtherPackages := [],
+  NeededOtherPackages := [ [ "GAPDoc", ">= 1.5" ] ],
```

Comment:
That's not wrong, but also not necessary, as GAPDoc is a required package and will always be present.